gips/datahandler/queue/rq.py

```diff
@@ -18,9 +18,35 @@ def get_job_name():
         raise queue.NoCurrentJobError('rq.get_current_job() returned None; no current job found')
     return job.id
 
+def is_job_alive(job_id):
+    """Tells whether the given job is currently running.
 
-def is_job_alive(sched_id):
-    raise NotImplementedError('NO support for this yet')
+    Job metadata in RQ has a TTL, so if no job matches the job ID,
+    False is returned because it's assumed the ID matched a job whose
+    metadata has since been removed from Redis.
+    """
+    job = get_queue().fetch_job(job_id)
+    return job is not None and job.is_started
```

Comment:
I don't think this will work as intended (probably due to my poor naming of `is_job_alive`). This is used in a check to make sure the job is still going, so a `queued` state should also be considered "alive"